src/helpers/user.helpers.js

```diff
@@ -14,13 +15,13 @@ function comparePassword(plainPassword, hashedPassword) {
 }
 
 function generateToken(payload, expiresIn) {
-  var token = jwt.sign(payload, process.env.SECRETE, { expiresIn });
+  const token = jwt.sign(payload, process.env.SECRETE, { expiresIn });
   return token;
 }
 
-// function decodeToken(token) {
-// 	const verify = jwt.verify(token, process.env.SECRETE)
-// 	return verify
-// }
+function decodeToken(token) {
+  const verify = jwt.verify(token, process.env.SECRETE);
+  return verify;
+}
 
-module.exports = { hashPassword, comparePassword, generateToken };
+module.exports = { hashPassword, comparePassword, generateToken, decodeToken };
```

Comment:
Use ES6

Reply:
The roles and the hashPassword functions are being used in models and seeders files which when running with sequelize they do not use babel so thereby requiring to use es5 files even in import